torch/_inductor/fx_passes/quantization.py

```diff
@@ -389,6 +389,31 @@ def qlinear(match: Match, *args, **kwargs):
     return qlinear
 
 
+def _is_valid_quantized_conv_binary_optimization_pattern(output_dtype):
```

Comment:
Add comment to briefly describe the rules being checked here?

Reply:
Thanks and added the comment.